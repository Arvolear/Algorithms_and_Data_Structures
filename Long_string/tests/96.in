70648946377331
64186580