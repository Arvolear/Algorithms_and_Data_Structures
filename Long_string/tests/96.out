70649010563911
70648882190751
4534714248564266417980
1100680